e2e_samples/dataset_versioning/datafactory/README.md

```diff
@@ -0,0 +1,22 @@
+# Overview
+
+This folder is for Data pipeline which copy data from SQL DB to Azure Storage (Delta Lake). We are using Azure Data Factory to implement pipeline.
+
+## How to deploy it
+
+1. Provision Azure Resources by IaC (Terraform)
+1. [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+1. Run command to deploy ([document](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deploy-cli#deploy-local-template-or-bicep-file))
+
+```bash
+az deployment group create --name {Your deployment name. Anyname is OK} --resource-group {Your resource group name} --template-file ./arm_template.json --parameters factoryName="{Your data factory name}" KeyVault_properties_typeProperties_baseUrl="{Your key vault url}" AzureBlobFS_properties_typeProperties_serviceEndpoint="{Your blob storage url}"
+```
+
+### Parameters in deployment script
+
+|Name|Description|
+|--|--|
+|factoryName|Azure Data Factory name where you'll deploy ARM template into|
```

Comment:
We discussed in chat but it's not very clear how to get this information. Maybe have a better description or automate if possible.

Reply:
Have automated in this PR. A user can get deployment script when she/he provision infra successfully.
https://github.com/Azure-Samples/modern-data-warehouse-dataops/pull/305

Reply:
Have automated in https://github.com/Azure-Samples/modern-data-warehouse-dataops/pull/305